testColorToNumberMatcher.c

```diff
@@ -0,0 +1,21 @@
+#include "testColorToNumberMatcher.h"

```

Comment:
the name 'color to number' indicates only one way matching, while the content of the file does both